components/pagination/collections/paging_collection.js

```diff
@@ -377,6 +373,9 @@
                 }
             }
         });
+
+        var PagingCollectionProto = PagingCollection.prototype;
```

Comment:
Not to take the nitpick train too much further, but you could also just remove the `PagingCollectionProto` entirely :rocket: